modules/installation-about-restricted-network.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_restricted_networks/installing-restricted-networks-preparations.adoc
+
+[id="installation-about-restricted-networks_{context}"]
+= About installations in restricted networks
+
+In {product-title} {version}, you can perform an installation that does not
+require an active connection to the internet to obtain software components. You
+complete an installation in a restricted network on only infrastructure that you provision,
+not infrastructure that the installer provisions, so your platform selection is
+limited.
+// maybe point out that you can follow the bare metal installation rules on supported hardware and link to the matrix
+
+If you choose to perform a restricted network installation on a cloud platform, you
+still require access to its cloud APIs. Some cloud functions, like
+Amazon Web Service's IAM service, require internet access, so might still
```

Comment:
The last part ("...so might still require internet access") seems redundant. You could try either: 
- "Some cloud functions...still require internet access." (full stop)
- "...so _you_ might still require internet access."